shared/slackutil/messaging.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// messaging.go includes functions to send message to Slack channel.
+
+package slackutil
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"net/http"
+	"net/url"
+)
+
+const (
+	slackChatPostMessageURL = "https://slack.com/api/chat.postMessage"
+)
+
+// SlackOperations defines the operations that can be done to Slack
+type SlackOperations interface {
+	PostMessageToChannel(text, channel string) error
+}
+
+// slackClient contains Slack bot related information
+type slackClient struct {
```

Comment:
should this be called just client? Else, this will be slackutil.slackClient, which adds the stutter.

Reply:
Done.